src/hotspot/cpu/aarch64/continuationFreezeThaw_aarch64.inline.hpp

```diff
@@ -145,13 +146,11 @@ inline void FreezeBase::relativize_interpreted_frame_metadata(const frame& f, co
 
   // on AARCH64, we may insert padding between the locals and the rest of the frame
   // (see TemplateInterpreterGenerator::generate_normal_entry, and AbstractInterpreter::layout_activation)
-  // so we compute locals "from scratch" rather than relativizing the value in the stack frame, which might include padding,
-  // since we don't freeze the padding word (see recurse_freeze_interpreted_frame).
+  // since we freeze the padding word (see recurse_freeze_interpreted_frame) in order to keep the same relativized
```

Comment:
```suggestion
  // because we freeze the padding word (see recurse_freeze_interpreted_frame) in order to keep the same relativized
```
... for clarity.